build/package-exclude.txt

```diff
@@ -23,5 +23,6 @@ applications/dashboard/styleguide/**
 applications/dashboard/template/**
 applications/dashboard/scss/**
 applications/dashboard/js/src/**
+dist/knowledge/**
```

Comment:
Shouldn't this build to be knowledge agnostic?

Reply:
We had a discussion in person. Theses are files from the forum targeting the KB search page (the forum category filter input). We may revisit in the future.